tasks/ItemNFT.tx.js

```diff
@@ -90,12 +90,27 @@ task("itemnft:mint-batch", "mint tokens")
   )
   .addOptionalParam("password", "password for decrypting wallet")
   .addParam("to", "The target address receiving minted token")
-  .addParam("ids", "Comma seperated token ids")
+  .addOptionalParam("ids", "Comma seperated token ids")
+  .addOptionalParam("range", "start-end formatted range")
   .setAction(async (taskArgs) => {
     printArguments(taskArgs);
+
+    let ids;
+    if (taskArgs.ids) {
```

Comment:
How about outputting an error if neither args are entered?
Currently, it looks like ids will run with an empty array in this case.

Reply:
Fixed. Thanks. 👍